### DISCUSSION TITLE
Preview Modus

### DISCUSSION THREAD

**Discussion by bacanol** [butterwecksolutions/Spotmarket-Switcher#164] [Ideas]
Moin Christian,

ich finde den Spotmarket-Swichter echt gelungen und nutze ihn schon eine ganze Weile. Mir ist schon häufiger die Idee gekommen, dass ein Preview Modus hilfreich wäre beim feinjustierten von Parametern. Ich stelle mir das so vor, dass man beim Aufruf eine Uhrzeit mitgeben kann, um dann die Logik zu dem angegebenen Zeitpunkt zu testen. Ein echtes Laden soll dann nicht stattfinden. So könnte man nach Anpassungen der Werte simulieren, was er dann ggf Nachts macht oder auch nicht. 

Was meinst Du zu der Idee und dem Thema?

Vielen Dank und viele Grüße

**Comment by butterwecksolutions** [comment]
Hi, Danke für Dein Feedback und die Anregung.
Momentan entwickeln wir wesentliche Verbessungen wie z.B. die dynamische Steuerung des ESS. 
Der Akku wird dann nur noch bei höchsten Preisen entladen falls er nicht voll genug ist.
Ich schaue mir das mit dem Testzeitpunkt an wenn wir mit den größeren Baustellen durch sind.

Gruß!
Christian

**Comment by butterwecksolutions** [comment]
@bacanol Schau Dir mal die neue 2.4-DEV an. https://github.com/christian1980nrw/Spotmarket-Switcher/tree/dev
In der Beschreibung ist schon ein neuer Screenshot davon.
Dort gibt es nun eine Vorschau im Log, zu welchem Preis ausgelöst wird. Das sollte in die Richtung gehen die Du meinst.
Da das Verhalten nun nicht nur von der Uhrzeit, sondern nun auch vom SOC und vom Wetter abhängig ist,
tue ich mir schwer mir einen echten Vorschaumodus dafür auszudenken.

Gruß!
Christian

**Comment by butterwecksolutions** [comment]
@bacanol Ich schließe das hier mal. Es gibt ja nun die Vorschau im Log zu welchen Preisen etwas getan wird. Eine andere einfache Möglichkeit ist z.B. testweise die Systemzeit zu verstellen um Dinge zu simulieren oder nachzustellen.

Viele Grüße!
